### PR TITLE
MM-22013 - Allow users to specify default download locations

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/desktop/pull/1383

Updated:
- User's Guide > Getting Started > Using Mattermost Desktop App > App Options
   - Created a new subsection called "Download Location" and documented how to change where files are saved to the local machine

### CODE REVIEW

**Review by esethna — APPROVED** [review]

**Comment by esethna** [comment]
Probably don't need a dev review on end user docs unless there's a technical change to review

File: source/help/apps/desktop-guide.rst [hunk 1]
```diff
@@ -27,36 +27,36 @@ Adding Servers
...
 ~~~~~~~~~~~~~~~~
 
 To remove a server from your desktop app environment:
 
-1. On Windows, go to **... > File > Settings**. On Mac, go to **Mattermost > Preferences**.
-2. Next to the server or team that you want to remove, click **Remove**.
-3. Click **Remove**.
+1. On Windows, go to **... > File > Settings**. On Mac or Linux, go to **Mattermost > Preferences**.
+2. Next to the server or team that you want to remove, select **Remove**.
+3. select **Remove**.
```
Comment (justinegeffen) [inline source/help/apps/desktop-guide.rst:52]:
```suggestion
3. Select **Remove**.
```

File: source/help/apps/desktop-guide.rst [hunk 2]
```diff
@@ -27,36 +27,36 @@ Adding Servers
...
 
 Removing Servers
 ~~~~~~~~~~~~~~~~
 
 To remove a server from your desktop app environment:
 
-1. On Windows, go to **... > File > Settings**. On Mac, go to **Mattermost > Preferences**.
-2. Next to the server or team that you want to remove, click **Remove**.
-3. Click **Remove**.
+1. On Windows, go to **... > File > Settings**. On Mac or Linux, go to **Mattermost > Preferences**.
```
Comment (justinegeffen) [inline source/help/apps/desktop-guide.rst:50]:
```suggestion
1. On Windows, go to **... > File > Settings**. On macOS or Linux, go to **Mattermost > Preferences**.
```

File: source/help/apps/desktop-guide.rst [hunk 3]
```diff
@@ -27,36 +27,36 @@ Adding Servers
...
+4. Select **Add**.
 
 Editing Servers
 ~~~~~~~~~~~~~~~
 
 To edit a server in your desktop app environment:
 
-1. On Windows, go to **... > File > Settings**. On Mac, go to **Mattermost > Preferences**.
-2. Next to the server you want to update, click **Edit**.
+1. On Windows, go to **... > File > Settings**. On Mac or Linux, go to **Mattermost > Preferences**.
```
Comment (justinegeffen) [inline source/help/apps/desktop-guide.rst:40]:
```suggestion
1. On Windows, go to **... > File > Settings**. On macOS or Linux, go to **Mattermost > Preferences**.
```

File: source/help/apps/desktop-guide.rst [hunk 4]
```diff
@@ -27,36 +27,36 @@ Adding Servers
...
+2. Next to the server or team that you want to remove, select **Remove**.
+3. select **Remove**.
 
 App Options
 -----------
 
 In addition to `Mattermost Account Settings <https://docs.mattermost.com/help/settings/account-settings.html>`__, the Desktop App provides additional options to customize your experience.
 
-The options appear in the Settings page, available on Windows from **... > File > Settings**, or on Mac from **Mattermost > Preferences**.
+The options appear on the Settings page, available on Windows from **... > File > Settings**, or on Mac or Linux from **Mattermost > Preferences**.
```
Comment (justinegeffen) [inline source/help/apps/desktop-guide.rst:59]:
```suggestion
The options appear on the Settings page, available on Windows from **... > File > Settings**, and on macOS or Linux from **Mattermost > Preferences**.
```

File: source/help/apps/desktop-guide.rst [hunk 5]
```diff
@@ -85,6 +85,15 @@ Leave app running in notification center when application window is closed (Linu
 Use GPU hardware acceleration
     When enabled, the Mattermost user interface is rendered more efficiently but can lead to decreased stability for some systems. This setting is enabled by default. You must restart the app to change this setting.
     
+Download Location
+~~~~~~~~~~~~~~~~~
+
+You can specify where files downloaded from the Mattermost desktop app are saved on your local machine. By default, files are saved to the ``Downloads`` folder.
+
+1. On Windows, go to **... > File > Settings**. On Mac or Linux, go to **Mattermost > Preferences**.
```
Comment (justinegeffen) [inline source/help/apps/desktop-guide.rst:93]:
```suggestion
1. On Windows, go to **... > File > Settings**. On macOS or Linux, go to **Mattermost > Preferences**.
```

**Review by justinegeffen — APPROVED** [review]
Some minor suggestions for naming but otherwise LGTM and approving to unblock.

**Comment by amyblais** [comment]
@cwarnermm There are some merge conflicts here, can you help resolve and then we can merge this?

**Comment by amyblais** [comment]
@cwarnermm Looks like there are merge conflicts again after I merged https://github.com/mattermost/docs/pull/4568. Sorry :(

**Pull request merged**